### ISSUE TITLE
keyboard lights not working, even keyboard not working.

### ISSUE THREAD

**Issue by Gabo-Tech** [slimbook/slimbookrgbkeyboard#10]
I'm using a titan, after the last firmware update the keyboard died for no apparent reason, I was able to reinstall the drivers and set it up, now keyboard works but with some misconfigurations on the layout and backlight doesn't work, when running the slimbookrgbkeyboard from terminal I get this error:
`device handle could not be acquired: no suitable device found`

Any ideas on how to troubleshoot it? no cool having a 'high end' laptop less tn 3 years that was almost 4k euros with an unusable keyboard just from a kernel update...

Randomly the keyboard stops working, and randomly too the lights get on... So the hardware is not damaged, I really take a lot of care of the laptop. 

Any hint would be helpful. I'm using latest version of Pop OS. and I also have the latest version of the BIOS.

**Comment by Lt-Henry** [comment]
Did the keyboard start to fail after firmware update or kernel update? Slimbook firmware upgrade is currently a manual and explicit process (fwupd is work in progress).

In case we are talking about a system upgrade, what Ubuntu version are you using? I have to double check it but I think we didn't updated Titan related drivers recently.

**Comment by Gabo-Tech** [comment]
Yes, was after a firmware update, these are the details of the system:
`Operating System: Pop!_OS 22.04 LTS               
          Kernel: Linux 6.9.3-76060903-generic
    Architecture: x86-64
 Hardware Vendor: SLIMBOOK
  Hardware Model: TITAN
`
